tests/writers.rs

```diff
@@ -111,32 +128,49 @@ impl World for WriterWorld {
             winning_writer_address: String::from(""),
 
             anvil: Option::None, // Anvil is started and stopped in before/after hooks. Better way to do this?
+            nft_contract: Option::None,
         })
     }
 }
 
 
-//#[tokio::main]
+#[tokio::main]
 // Test runner
-//fn main() -> eyre::Result<()> {
-fn main()  {
+async fn main() -> eyre::Result<()> {
 
     let world = WriterWorld::cucumber()
         // Start a fresh anvil before each scenario
         .before(move |_, _, _, world| {
             async move {
-                world.anvil = Option::Some(Anvil::new().spawn());
-                let endpoint = world.anvil.as_ref().unwrap().endpoint();
+                let anvil = Anvil::new()
+                    .fork("https://mainnet.infura.io/v3/bb3bdc9bf5c54df8998e697487bd03f9")
```

Comment:
Not blocking, assuming we want to take this out eventually, and will keep in our .env files or as GH secret